### ISSUE TITLE
TypeError: 'Pipe' object cannot be interpreted as an integer

### ISSUE THREAD

**Issue by JoshZastrow** [sspipe/sspipe#1]
Hi,

My pipelines built with sspipe often fail due to TypeError on one of the transformations. Even though each step returns a DataFrame object I still get this failure.

**Sample Code:**
```

import pandas as pd
from sspipe import px
from urllib.request import urlretrieve

# Get Data to work with
url = 'http://files.zillowstatic.com/research/public/Neighborhood/Neighborhood_MedianRentalPrice_AllHomes.csv'
local_path = './Neighborhood_MedianRentalPrice_AllHomes.csv'
urlretrieve(url, local_path)

zillow = pd.read_csv(local_path)

# This Works
zillow_regions =  zillow \
    .loc[zillow.State.isin(['CA', 'NY', 'WA', 'DC']), ['RegionName', 'City', 'State']] \
    .sort_values(['City', 'RegionName']) \
    .drop_duplicates()

# This Does Not Work
zillow_regions = (
    zillow
    | px.loc[px.State.isin(['CA', 'NY', 'WA', 'DC']), ['RegionName', 'City', 'State']]
    | px.sort_values(['City', 'RegionName'])
    | px.drop_duplicates()
)
zillow_regions.head()
```

**Error Message when using sspipe:**
```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
TypeError: 'Pipe' object cannot be interpreted as an integer
Exception ignored in: 'pandas._libs.tslibs.util.is_period_object'
TypeError: 'Pipe' object cannot be interpreted as an integer
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
TypeError: 'Pipe' object cannot be interpreted as an integer
Exception ignored in: 'pandas._libs.lib.is_interval'
TypeError: 'Pipe' object cannot be interpreted as an integer
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
TypeError: 'Pipe' object cannot be interpreted as an integer
Exception ignored in: 'pandas._libs.tslibs.util.is_offset_object'
TypeError: 'Pipe' object cannot be interpreted as an integer
```

**Comment by mhsekhavat** [comment]
Thanks for your bug report @JoshZastrow!

 I believe it is now fixed in version 0.1.7. Please let me know if the issue is still not resolved.

**Comment by JoshZastrow** [comment]
It's working now, thanks for the fix!
